web/public/js/directives/cd-action-bar/index.js

```diff
@@ -38,6 +45,16 @@
           $footer.off('inview');
           $element.remove();
         };
+
+        ctrl.$onChanges = function (changes) {
+          if (changes.forceFixed) {
```

Comment:
`changes.forceFixed` is an object from Angular. If `forceFixed` has been changed, `changes.forceFixed` will always be an object. `changes.forceFixed.currentValue` would be null, not `changes.forceFixed`.